examples/seismic/acoustic/accuracy.ipynb

```diff
@@ -242,32 +303,42 @@
     "plt.savefig('wavefieldperf.pdf')\n",
```

Comment:
No need to save I guess.

Reply:
I guess I think the outputs are useful, especially if you want to grab an image. But happy to remove if there is consensus.

Reply:
I agree. Usually we just plot the images, but not save them.

Reply:
need clarification: keep the output (from original nb) or remove?

Reply:
Saving doesn't hurt no?

Reply:
It doesn't hurt, but what's the point? I mean we could save stuff in other tutos as well but I guess the point for the user is to see the plot rather that save it. In any case, just my 2 cents.

Reply:
It reproduces the plots in the paper

Reply:
Good. Did not know that.

Reply:
resolving ...